content/articles/Integrating-mongodb-with -java/index.md

```diff
@@ -0,0 +1,241 @@
+### Integrating MongoDB With Java
+
+MongoDB is a NoSQL database that stores record as documents. It uses JSON syntax and JavaScript Object Notation to store data. When integrating MongoDB with Java, it uses a different syntax.
+In this article, you'll learn how to connect MongoDB with Java using Eclipse and how to perform CRUD (create, read, update, and delete) operations using Java and the MongoDB Java Driver. MongoDB is based on a document model written in C++, JavaScript, and Python.
+
+### Table of Contents 
+- [Prerequisites](#prerequisite) 
+- [Relational Database Management Systems vs MongoDB](#Relational-Database-Management-Systems-vs-MongoDB)
+-  [Connecting MongoDB with Java](#Connecting-MongoDB-with-Java) 
+- [Creating MongoDB collection](#Creating-MongoDB-collection)
+- [Insert data into the collection](#Insert-data-into-the-collection) 
+- [Update data in the collection](#Update-data-in-the-collection)
+- [Delete data in the collection](#Delete-data-in-the-collection) 
+- [Delete the collection](#Delete-the-collection)
+- [Eclipse versus Intellij Idea in MongoDB integration with Java](#Eclipse-versus-Intellij-Idea-in-MongoDB-integration-with-java)  

```

Comment:
`(#Relational-Database-Management-Systems-vs-MongoDB)`

In this part, after `#`, all the letters must be in lower case separated  by `-`.

Ensure that the topic name is same as this.

Use VS code markdown extensions to ensure if the hyperlinks is working as expected.